Doc/s5/ep2008/stupidlowercase.py

```diff
@@ -1,4 +1,3 @@
-
 def stupid_lower_case(char* s):
```

Comment:
My feeling is that Doc/s5/ep2008 is an archive of an old europython presentation and should probably just be left historical